src/main/java/mate/controller/DeleteCarsDriverController.java

```diff
@@ -0,0 +1,28 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class DeleteCarsDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String carId = req.getParameter("car_id");
+        carService.removeDriverFromCar(
+                driverService.get(Long.parseLong(req.getParameter("driver_id"))),
+                carService.get(Long.parseLong(carId))
+        );
```

Comment:
The same - keep your code readable, no need to do it in 1 line, extract it into local variables